lib/template-mixins.js

```diff
@@ -84,22 +84,30 @@ module.exports = function (fields, options) {
             return translate(sharedTranslationsKey + key);
         };
 
-        var conditionaltranslate = function (key) {
+        // Like t() but returns null on failed translations
+        var conditionalTranslate = function (key) {
             key = sharedTranslationsKey + key;
             var translated = translate(key);
             return translated !== key ? translated : null;
         };
 
+        var getTranslationKey = function (key, property) {
+            return fields && fields[key] ? fields[key][property] : 'fields.' + key + '.' + property;
```

Comment:
How about if we also test for fields[key][property]?

If I am understanding correctly, the original code also suffered from the shortcoming you describe.